src/_includes/docs/install/devices/ios-physical.md

```diff
@@ -1,13 +1,3 @@
-:::warning
-An upcoming change to iOS has caused a temporary break in Flutter's debug mode
-on physical devices running iOS 18.4 (currently in beta).
-If your physical device is already on iOS 18.4, we recommend switching to the
-**Virtual device** tab and following the instructions for using a simulator.
-See [Flutter on latest iOS][] for details.
-:::
-
-[Flutter on latest iOS]: /platform-integration/ios/ios-latest
-
```

Comment:
I removed the warning entirely from the getting started page as it seems unlikely someone will start iOS development using the non-latest beta.